src/spec-helpers.ts

```diff
@@ -47,6 +48,7 @@ const baseTestModuleMetadata: TestModuleMetadata = {
     RouterTestingModule,
     HttpClientTestingModule,
     ReactiveFormsModule,
+    BrowserAnimationsModule,
```

Comment:
Wird das wirklich benötigt?

Reply:
Hast du diesen Fehler nicht in der Konsole, wenn du die Tests laufen lässt?

`ERROR: 'ERROR', Error: Found the synthetic property @flyInOut. Please include either "BrowserAnimationsModule" or "NoopAnimationsModule" in your application.`

Dieser Change hat bei mir das Problem behoben. Aber ich war nicht ganz sicher, ob das die richtige Lösung ist...